charts/kserve-resources/templates/clusterrole.yaml

```diff
@@ -307,27 +169,12 @@ rules:
 - apiGroups:
   - serving.kserve.io
   resources:
+  - clusterservingruntimes/status
+  - inferencegraphs/status
+  - inferenceservices/status
+  - servingruntimes/status
   - trainedmodels/status
   verbs:
   - get
   - patch
   - update
-
```

Comment:
Can we add a flag to enable/disable this role ? I think currently it is still used with Kafka eventing

Reply:
Added